tests/core/node.test.ts

```diff
@@ -4,6 +4,7 @@ import { CCObject } from "../../cocos/core";
 import { NodeEventType } from "../../cocos/scene-graph/node-event";
 import { ccclass } from "../../cocos/core/data/decorators";
 import {director, game } from '../../cocos/game';
+import { boolean } from "yargs";
```

Comment:
Why to import `boolean` from 'yargs'?

Reply:
I don't know, VS code do it automatically.

Reply:
I think it could be removed.